eox_tagging/api/v1/viewset.py

```diff
@@ -204,10 +204,10 @@ def create(self, request, *args, **kwargs):
         """
 
         @audit_method(action="eox_tagging-api-v1-viewset:tagviewset-create")
-        def audited_create(headers, body):  # pylint: disable=unused-argument
+        def audited_create(body):  # pylint: disable=unused-argument
             return super(TagViewSet, self).create(request, *args, **kwargs)
 
-        return audited_create(headers=request.headers, body=request.data)
```

Comment:
What are the elements that are not serializable? I thought all the elements were strings. I think the problem is headers is not a real dict, so maybe something like `dict(request.headers)` could work.

Reply:
I will test that

Reply:
The problem is not the headers, the headers variable is a dict the problem is one element inside the dict which is not a dict, with your suggestion is the same

Reply:
In that case go ahead